### PR TITLE
Order of layers is updated correctly in CartoDB.js when a layer is created from an analysis

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Fixes https://github.com/CartoDB/cartodb/issues/8245

@viddo 👍 ? (I could have done more testing, but I feel this is good enough given the amount of work we have now)

### CODE REVIEW

File: lib/assets/test/spec/cartodb3/deep-insights-integrations.spec.js [hunk 1]
```diff
@@ -296,114 +314,6 @@ describe('deep-insights-integrations', function () {
       });
     });
 
-    describe('.infowindow', function () {
```
Comment (alonsogarciapablo) [inline lib/assets/test/spec/cartodb3/deep-insights-integrations.spec.js]: These tests were duplicated. cc: @matallo

**Comment by viddo** [comment]
👍

**Comment by saleiva** [comment]
can we merge? useful for passing the smoke tests

**Comment by alonsogarciapablo** [comment]
Will deploy this now. Thanks guys!

**Comment by alonsogarciapablo** [comment]
@saleiva, my friend, this should be working now 👍

**Comment by saleiva** [comment]
Looks like that :_)

**Comment by saleiva** [comment]
THANKS!

**Pull request merged**